view/adminhtml/templates/emailtemplate/script.phtml

```diff
@@ -0,0 +1,19 @@
+<?php
+/* @var $block Magento\Framework\View\Element\Template */
+$viewModel = $block->getViewModel();
+?>
+
+<?php if ($viewModel->isNewTemplate()) { ?>
```

Comment:
use if condition like if(): & then end it with endif; instead of bracket. That makes easy to read if condition within html code